pkg/controller/core/localqueue_controller.go

```diff
@@ -275,21 +309,29 @@ func (r *LocalQueueReconciler) UpdateStatusIfChanged(
 	reason, msg string,
 ) error {
 	oldStatus := queue.Status.DeepCopy()
-	pendingWls, err := r.queues.PendingWorkloads(queue)
-	if err != nil {
-		r.log.Error(err, failedUpdateLqStatusMsg)
-		return err
-	}
-	stats, err := r.cache.LocalQueueUsage(queue)
-	if err != nil {
-		r.log.Error(err, failedUpdateLqStatusMsg)
-		return err
+	if ptr.Deref(queue.Spec.StopPolicy, kueue.None) == kueue.None {
+		pendingWls, err := r.queues.PendingWorkloads(queue)
+		if err != nil {
+			r.log.Error(err, failedUpdateLqStatusMsg)
+			return err
+		}
+		stats, err := r.cache.LocalQueueUsage(queue)
+		if err != nil {
+			r.log.Error(err, failedUpdateLqStatusMsg)
+			return err
+		}
+		queue.Status.PendingWorkloads = pendingWls
+		queue.Status.ReservingWorkloads = int32(stats.ReservingWorkloads)
+		queue.Status.AdmittedWorkloads = int32(stats.AdmittedWorkloads)
+		queue.Status.FlavorsReservation = stats.ReservedResources
+		queue.Status.FlavorUsage = stats.AdmittedResources
+	} else {
+		queue.Status.PendingWorkloads = 0
+		queue.Status.ReservingWorkloads = 0
+		queue.Status.AdmittedWorkloads = 0
```

Comment:
This is not accurate. If the policy is just Hold, we still need to track what is still running.

Reply:
Fixed